### PR TITLE
Integrate documentation

### PR DESCRIPTION
**Pull request by mchv**
- Clean unused files
- Integrate documentation

screen shot 2014-07-07 at 16 38 45
screen shot 2014-07-07 at 16 38 57
screen shot 2014-07-07 at 16 39 07
screen shot 2014-07-07 at 16 39 17

The content in the documentation may not be perfect, but it will be easier to update once this pull request has been merged.

### CODE REVIEW

File: assets/js/documentation.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            updateContent(html);
+        }
+    }
+
+    function updateContent(html, common) {
+            
+        var name = $('#doccontent').attr('data-file');
+
+        $('#doccontent').empty();
+        $('#doccontent').append('<div class="ui ribbon label"><a href="https://github.com/guardian/content-api-docs/edit/gh-pages/docs/' + name + '.md">Edit me <i class="github alternate large icon"></i></a></div>');
```
Comment (gidsg) [inline assets/js/documentation.js]: These 'Edit me' links 404 now, I think we should just delete them.

File: documentation/index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	<div class="main">
+	    <ul class="nav">
+            <li><a href="../explore">Explore content</a></li>
+            <li><a href="../access">Access API</a></li>
+            <li><a class="active" href="../documentation">Read documentation</a></li>
+        </ul>
+
+        <div class="pushed">
+            <div id="sidebar" class="ui inverted vertical menu sidebar active">
+                <a class="item" href="index.html" >General</a>
```
Comment (gidsg) [inline documentation/index.html]: I find 'General' vague. Can we call it 'Getting Started' to be consistent with the page heading?

Comment (gidsg) [inline documentation/index.html]: Can you just link to `/` instead? I don't really want links floating around to `/documentation` and `/documentation/index.html`

File: documentation/md/index.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+  .ir a:after, a[href^="javascript:"]:after, a[href^="#"]:after { content: ""; }
+  pre, blockquote { border: 1px solid #999; padding-right: 1em; page-break-inside: avoid; }
+  tr, img { page-break-inside: avoid; }
+  img { max-width: 100% !important; }
+  @page :left { margin: 15mm 20mm 15mm 10mm; }
+  @page :right { margin: 15mm 10mm 15mm 20mm; }
+  p, h2, h3 { orphans: 3; widows: 3; }
+  h2, h3 { page-break-after: avoid; }
+}
+</style><title>index</title></head><body><h1 id="getting-started">Getting started</h1>
```
Comment (gidsg) [inline documentation/md/index.html]: Can you make these `<h2>` instead? You are recommended to have only 1 `<h1>` per page.

File: public/css/Guardian Egyp Web-Regular.svg [hunk 4]
```diff
@@ -1,358 +0,0 @@
-<?xml version="1.0" standalone="no"?> <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd" > <svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%">
```
Comment (gidsg) [inline public/css/Guardian Egyp Web-Regular.svg:1]: Are we actually using the Guardian fonts? If not we should remove them, also if we are there are some licensing issues we have to consider.

File: documentation/index.html (hunk 2 above)
Comment (mchv) [inline documentation/index.html]: @gidsg I have updated to 'Getting Started' but we can't use '/' there.

File: public/css/Guardian Egyp Web-Regular.svg (hunk 4 above)
Comment (nicl) [inline public/css/Guardian Egyp Web-Regular.svg:1]: @gidsg what are the licensing issues?

Comment (mchv) [inline public/css/Guardian Egyp Web-Regular.svg:1]: The commit removes them.

File: .gitignore [hunk 5]
```diff
@@ -1,2 +1 @@
-.DS_STORE
-
+.DS_Store
```
Comment (gidsg) [inline .gitignore:1]: Can we just do

```
.*
!/.gitignore`
```

to ignore all dotfiles?

**Comment by gidsg** [comment]
:+1: once comments addressed.

File: .gitignore (hunk 5 above)
Comment (mchv) [inline .gitignore:1]: Fine to me, let's do this outside of the PR.

**Pull request merged**